From bcdf84b4889777daf9ebb73a7d7e25c974f210e8 Mon Sep 17 00:00:00 2001
From: Xylar Asay-Davis <xylarstorm@gmail.com>
Date: Fri, 27 Jan 2023 14:12:11 -0700
Subject: [PATCH] Make `setup.py` consistent with conda recipe

Inconsistencies in `setup.py` are causing `pip check` to fail
on the conda package.
---
 setup.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 0b94541..b58033c 100644
--- a/setup.py
+++ b/setup.py
@@ -7,10 +7,11 @@
     author_email="forsyth2@llnl.gov, golaz1@llnl.gov, shaheen2@llnl.gov",
     description="Long term HPSS archiving software for E3SM",
     packages=find_packages(include=["zstash", "zstash.*"]),
+    python_requires=">=3.6",
     install_requires=[
-        "six==1.16.0",
-        "globus-sdk<4.0.0>=3.0.0",
-        "fair-research-login==0.2.6",
+        "six",
+        "globus-sdk>=3.0.0,<4.0.0",
+        "fair-research-login",
     ],
     entry_points={"console_scripts": ["zstash=zstash.main:main"]},
 )